Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.4.1
+
+LABEL maintainer Travis CI GmbH <support+travis-app-docker-images@travis-ci.com>
+
+RUN apt-get update && apt-get upgrade -y --no-install-recommends && apt-get install -y postgresql
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY Gemfile      /usr/src/app
+COPY Gemfile.lock /usr/src/app
+
+RUN bundle install --deployment
+
+COPY . /usr/src/app
+
+RUN apt-cache search postgres
+# Install sqitch so migrations work
+RUN env BINDIR=/usr/local/bin /usr/src/app/script/install-sqitch
```

Comment:
Back from house things! (well, sorta) I'm having issues getting sqitch to install and work properly in the Docker environment. Thought it was just path issues but now even logging in and running it seems it's having issues finding module? Dunno. Also wanted to install maybe partman but figured first things first. :) 

Have any time to see about getting the sqitch migrations and such to work in docker? (If we get that working I may end up using logs in a container even if the rest of 2.2 doesn't use the new containers).  I also have the beginnings of a docker compose project if you want to see what I'm doing there (it's a mess but getting better). Hit me up in Slack if you want that link (since it's still private).